tokens/style-dictionary.js

```diff
@@ -80,45 +84,41 @@ const colorTransform = (token, theme) => {
  * 2. 'theme' to output only theme's variables (e.g, 'light' or 'dark'), if theme is not provided - only
  * core tokens are built.
  */
-const createCustomCSSVariables = async ({
-  formatterArgs,
-  themeVariant,
-}, StyleDictionary) => {
-  // eslint-disable-next-line import/no-unresolved
-  const { sortByReference, usesReferences, getReferences } = (await import('style-dictionary/utils'));
+const createCustomCSSVariables = async ({ formatterArgs }) => {
+  const { fileHeader, formattedVariables } = await getStyleDictionaryUtils();
   const { dictionary, options, file } = formatterArgs;
-  const outputTokens = themeVariant
-    ? dictionary.allTokens.filter(token => token.filePath.includes(themeVariant))
-    : dictionary.allTokens;
-
-  const variables = outputTokens.sort(sortByReference(dictionary)).map(token => {
-    let { $value } = token;
-
-    const outputReferencesForToken = (token.original.outputReferences === false) ? false : options.outputReferences;
-
-    if (usesReferences(token.original.$value) && outputReferencesForToken) {
-      const refs = getReferences(token.original.$value, dictionary.tokens);
-      refs.forEach(ref => {
-        $value = $value.replace(ref.$value, `var(--${ref.name})`);
-      });
-    }
-
-    return `  --${token.name}: ${$value};`;
-  }).join('\n');
-
-  return `${createCustomHeader(StyleDictionary, file).join('\n')}\n:root {\n${variables}\n}\n`;
+  const { outputReferences, formatting } = options;
+  const variables = formattedVariables({
```

Comment:
[inform] Relies on helper function `formattedVariables` provided by `style-dictionary/utils` to format tokens as CSS variables instead of re-implementing much of its same logic here.